main.go

```diff
@@ -51,6 +52,7 @@ var (
 	gceVM                    = flag.String("gce-vm-experimental", "", "GCE VM name to use, instead of reading it from the metadata server. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
 	configMesh               = flag.String("config-mesh-experimental", "", "Dictates which Mesh resource to use. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
 	includeFederationSupport = flag.Bool("include-federation-support-experimental", false, "whether or not to generate configs required for xDS Federation. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
+	includeC2PAuthority      = flag.Bool("include-c2p-authority-experimental", false, "whether or not to include c2p TD authority for xDS Federation. Ignored if not used with include-federation-support-experimental flag. This flag is EXPERIMENTAL and may be changed or removed in a later release.")
```

Comment:
`c2p` might not be very descriptive for the users. Maybe `include-directpath-authority-experimental`. Wdyt @ejona86 ?

Reply:
meh. Users should never need to set this. It'll just become the default and then go away.